src/utils/composeReducers.js

```diff
@@ -0,0 +1,12 @@
+import mapValues from '../utils/mapValues';
+import pick from '../utils/pick';
+
+export default function composeReducers(reducers) {
+  const finalReducers = pick(reducers, (val) => typeof val === 'function');
+
+  return function Composition(atom = {}, action) {
```

Comment:
A few questions:
1. Why are you calling this "atom" and not "state" or "slice"?
2. Why does atom have a default value?
3. why is Composition capitalized?